modrequest.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+    "fmt"
+
+    "github.com/bwmarrin/discordgo"
+)
+
+const modRequest = "<@%s> requested a moderator in <#%s> here: %s\nReason: %s"
+
+func (b *bot) requestMod(s *discordgo.Session, m *discordgo.Message, parts []string) {
+    url := composeMessageURL(m)
+    s.ChannelSendMessage(feedChannel, fmt.Printf(modRequest, m.Author.ID, m.ChannelID, url, parts)
```

Comment:
`fmt.Printf` will print to stdout. You want `fmt.Sprintf` so that it returns a `string`.

Also, `parts` is an array, which cannot be processed by the format string `%s`. You want to pass in [`strings.Join(parts)`](https://golang.org/pkg/strings/#Join) instead. You'll also need to import `strings`.

```suggestion
    s.ChannelSendMessage(feedChannel, fmt.Sprintf(modRequest, m.Author.ID, m.ChannelID, url, strings.Join(parts, " "))
```

----

The above change combined with the suggestion in  will look like:

```suggestion
    b.discord.ChannelSendMessage(feedChannel, fmt.Sprintf(modRequest, m.Author.ID, m.ChannelID, url, strings.Join(parts, " "))
```